normalView/normalView.js

```diff
@@ -0,0 +1,34 @@
+let button = document.getElementById("button1");
+let notification="on";
+
+localStorage.getItem("notification",notification);
```

Comment:
I see the issue here, the returned value is not being saved to any variable. Also to get the storage item it only takes an argument, the name of item. ``` const notification = localStorage.getItem("notification");```
It should be declared inside onClick scope, otherwise the notification constant won't be accessible.

Note: What happens if the localStorage doesn't have a notification setting (eg. the use never used the app before). We can drop to a default, but also could be interesting to have a setting.js background file that sets defaults if they don't exist. I'll add it to the background.js file.
Again, 'notification' could be called 'pauseNotifications' :)